extensions/amp-sticky-ad/1.0/amp-sticky-ad.js

```diff
@@ -57,6 +63,33 @@ class AmpStickyAd extends AMP.BaseElement {
   buildCallback() {
     this.viewport_ = this.getViewport();
     this.element.classList.add('i-amphtml-sticky-ad-layout');
+
+    // For iPhone users with a home bar, we set a padding to prevent the ads from being unclickable.
+    // All current all-screen iPhones have an aspect ratio of ~2.16
+    if (
+      navigator.userAgent.match(/iPhone/i) &&
+      Number(window.screen.height) / window.screen.width > 2.1 &&
```

Comment:
I don't have a better solution but this seems brittle, hopefully we can remember to update this when new iPhone ratio comes out :)